Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -23,12 +24,12 @@ public class ActorPhysics : ActorComponent
 	[ReadOnly, SerializeField]
 	ActorStates _currentState = ActorStates.Jumping;
 	ActorStateMethod CurrentStateMethod;
-	public Dictionary<ActorStates, ActorStateMethod> stateMethodMap = new Dictionary<ActorStates, ActorStateMethod>();
+	Dictionary<ActorStates, ActorStateMethod> _stateMethodMap = new Dictionary<ActorStates, ActorStateMethod>();
 	#endregion
 
 	#region Movement
 	[Header( "Movement" )]
-	public Vector3 inputVec = Vector3.zero;
+	public Vector3 _inputVec = Vector3.zero;
```

Comment:
This shouldn't be underscored if it's public. You may want to use readonly here?